packages/tools/src/getLoginExpiration.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { getLoginExpiration } from './getLoginExpiration';
+
+describe('getLoginExpiration', () => {
+	it('should return 90 by default', () => {
+		expect(getLoginExpiration()).toBe(90);
+	});
+	it('should return 90 when value is 0', () => {
+		expect(getLoginExpiration(0)).toBe(90);
+	});
+	it('should return 90 when value is NaN', () => {
+		expect(getLoginExpiration(NaN)).toBe(90);
+	});
+	it('should return 90 when value is negative', () => {
+		expect(getLoginExpiration(-1)).toBe(90);
+	});
+	it('should return 90 when value is undefined', () => {
+		expect(getLoginExpiration(undefined)).toBe(90);
+	});
+	it('should return 90 when value is not a number', () => {
+		// @ts-expect-error - Testing
+		expect(getLoginExpiration('90')).toBe(90);
+	});
+	it('should return the value passed when its valid', () => {
```

Comment:
Do we need to add a validation for non-integer values?

Reply:
2 lines above, there's an string test 👀